concordium-node/src/stats_export_service.rs

```diff
@@ -67,11 +72,84 @@ impl prometheus::core::Collector for GrpcInFlightRequestsCollector {
     }
 }
 
+/// Collector tracking stats which needs to be synced with consensus on scrape
+/// through the FFI.
+pub struct StatsConsensusCollector {
+    /// Reference to consensus to support consensus queries.
+    consensus:              ConsensusContainer,
+    /// The baking committee status of the node for the current best block.
+    baking_committee:       IntGaugeVec,
+    /// Whether the node is a member of the finalization committee for the
+    /// current best block.
+    finalization_committee: IntGauge,
+}
+
+impl StatsConsensusCollector {
+    pub fn new(consensus: ConsensusContainer) -> anyhow::Result<Self> {
+        let baking_committee = IntGaugeVec::new(
+            Opts::new(
+                "consensus_baking_committee",
+                "The baking committee status of the node for the current best block, labelled \
+                 with the status where the only metric with value of 1 represents the current \
+                 status",
+            )
+            .variable_label("status"),
+            &["status"],
+        )?;
+
+        let finalization_committee = IntGauge::with_opts(Opts::new(
+            "consensus_finalization_committee",
+            "The finalization commmittee status of the node for the current best block. The \
+             metric will have a value of 1 when member of the finalization committee",
```

Comment:
As above, this is not for best block. It is for the current state of finalization.